inc/classes/optimization/class-remove-query-string.php

```diff
@@ -1,8 +1,9 @@
 <?php
```

Comment:
Let's move and rename this class file too.

Reply:
Moving this one will create conflicts with the changes we are making in 3.6, since it's deprecated in one of the opened PR. Shouldn't we leave it alone since the file is going to be deprecated anyway?

Reply:
But it won't be deprecated for at least 6 weeks. Until then, we can relocate it to bundle with the rest of the optimization files. Then we can merge `develop` into the 3.6 branch and update the PR that deprecates it.

Reply:
A good guideline is this: future releases need to stay up-to-date with the current work.; but not the other way around.

Reply:
Done in https://github.com/wp-media/wp-rocket/pull/2392/commits/23447dce7701dde57ad7f04d1803ef9774a46c49